docs/main/user-guides/migration/migration-using-pv-migrate.md

```diff
@@ -0,0 +1,303 @@
+---
+id: migration-using-pv-migrate
+title: Migration from Legacy Storage to Latest Storage Solution
+keywords:
+ - Migration
+ - Data Migration
+ - Migration from OpenEBS Local PV Device to OpenEBS LVM Local PV
+ - Local PV Device to Local PV LVM
+ - Local PV Device to Local PV ZFS
+ - Migration from OpenEBS cStor to OpenEBS Replicated
+ - cStor to Replicated
+ - cStor to Mayastor
+ - Jiva to Replicated
+ - Jiva to Mayastor
+description: This section outlines the process of migrating the legacy storage to latest storage solution.
+---
+
+# Migration using pv-migrate
+
+This section describes the process of migrating the legacy storage to latest storage solution.
+
+## Overview
+
+OpenEBS data migration is the process of transferring data from the legacy storage to the latest storage solution. The data can be migrated from any non-OpenEBS storage to OpenEBS storage and this section provides detailed description about the migration process for legacy storage users.
+
+The data migration of migrating the legacy storage to latest storage solution can be done in two ways:
+- [Migration using pv-migrate](#migration-using-pv-migrate)
+- [Migration using velero](../migration/migration-using-velero/)
```

Comment:
Why are we keeping Velero reference here? The plan is to have 3 pages - - Overview
- Data migration using pv-migrate utility
- Data migration using Velero utlity
Can we restructure the folders and files to match the above?

Reply:
I've restructured the folders.